Jenkinsfile

```diff
@@ -6,7 +6,7 @@ pipeline {
   environment {
     CI_RUN = "true"
     JDK_11 = 'openjdk@1.11.0-2'
-    JDK_12 = 'openjdk@1.12.0-1'
+    JDK_12 = 'openjdk@1.12.0-2'
```

Comment:
Btw. I learned afterwards that the `1.12.0` variant points to the latest available, maybe we use that?

See PRs here: https://github.com/shyiko/jabba/pull/498/files

Reply:
👍